Formula/grt.rb

```diff
@@ -0,0 +1,29 @@
+class Grt < Formula
+  desc "The Gesture Recognition Toolkit (GRT) for Real-time machine learning."
+  homepage "http://www.nickgillian.com/wiki/pmwiki.php/GRT/GestureRecognitionToolkit"
+  url "https://github.com/nickgillian/grt/archive/v0.0.1.tar.gz"
+  sha256 "56f90a9ffa8b2bf4e5831d39f9e1912879cf032efa667a5237b57f68800a2dda"
+
+  depends_on "cmake" => :build
+
+  def install
+    cd "build"
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
```

Comment:
Please remove this line. `make` is implied by `make install` and builds before installing unless there are issues with the build system (unlikely if it is generated by CMake).